gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -717,3 +730,274 @@ message TranslatedString {
   // The following extension IDs are reserved for private use by any organization.
   extensions 9000 to 9999;
 }
+
+// All messages below this comment are added in order to reflect the changes v3.1 of GTFS-ServiceChanges provide
+
+// Provides the updated properties for the trip.
+message TripProperties {
+  // Identifies a trip.
+  required string trip_id = 1;
+  // Dates on which the trip will be run in YYYYMMDD format.
+  required string start_date = 2;
+  // Departure start time of the trip when it’s duplicated.
+  required string start_time = 3;
+  // Refers to a route defined in the GTFS.
+  optional string route_id = 4;
+  // Text that appears on signage identifying the trip's destination to riders.
+  optional TranslatedString trip_headsign = 5;
+  // Public facing text used to identify the trip to riders.
+  optional TranslatedString trip_short_name = 6;
+  // Identifies the block to which the trip belongs.
+  optional string block_id = 7;
+  // Identifies a geospatial shape describing the vehicle travel path for a trip.
+  optional string shape_id = 8;
+}
+
+//  Provides the updated properties for the vehicle.
+message VehicleProperties {
+
+  // Indicates wheelchair accessibility.
+  optional WheelchairAccessibleStatus wheelchair_accessible = 1 [default = UNKNOWN_WHEELCHAIR_ACCESSIBILITY];
+  // Indicates whether bikes are allowed or not.
+  optional BikesAllowedStatus biked_allowed = 2 [default = UNKNOWN_BIKES_ALLOWANCE];
+
+  // Options for wheelchair_accessible:
+  // 0 or empty - No accessibility information for the trip.
```

Comment:
`or empty` should be removed - the default is explicitly stated with the field declaration